src/examples/welsh-language-toggle/example-welsh/index.njk

```diff
@@ -0,0 +1,14 @@
+---
+  layout: layout-example.njk
+---
+
+{%- from "hmrc/components/language-select/macro.njk" import hmrcLanguageSelect -%}
+
+{{
+  hmrcLanguageSelect({
```

Comment:
Can both urls be sent through so devs don't have to add conditional logic to send the inactive one?
Then an 'active' state can be set with another param.

Reply:
Have added examples for that